src/Knp/Menu/Twig/Helper.php

```diff
@@ -81,6 +86,99 @@ public function get($menu, array $path = array(), array $options = array())
      */
     public function render($menu, array $options = array(), $renderer =  null)
     {
+        $menu = $this->castMenu($menu);
+
+        return $this->rendererProvider->get($renderer)->render($menu, $options);
+    }
+
+    /**
+     * A string representation of this menu item
+     *
+     * e.g. Top Level > Second Level > This menu
+     *
+     * @param ItemInterface $item
+     * @param string        $separator
+     *
+     * @return string
+     */
+    public function pathAsString(ItemInterface $menu, $separator = ' > ')
+    {
+        if (null === $this->menuManipulator) {
+            throw new \BadMethodCallException('The menu manipulator must be set to transform a path to a string');
+        }
+
+        return $this->menuManipulator->getPathAsString($menu, $separator);
+    }
+
+    /**
+     * Renders an array ready to be used for breadcrumbs.
+     *
+     * Each element in the array will be an array with 3 keys:
+     * - `label` containing the label of the item
+     * - `url` containing the url of the item (may be `null`)
+     * - `item` containing the original item (may be `null` for the extra items)
+     *
+     * The subItem can be one of the following forms
+     *   * 'subItem'
+     *   * ItemInterface object
+     *   * array('subItem' => '@homepage')
+     *   * array('subItem1', 'subItem2')
+     *   * array(array('label' => 'subItem1', 'url' => '@homepage'), array('label' => 'subItem2'))
+     *
+     * @param mixed $item
+     * @param mixed $subItem A string or array to append onto the end of the array
+     *
+     * @return array
+     */
+    public function getBreadcrumbsArray($menu, $subItem = null)
+    {
+        if (null === $this->menuManipulator) {
+            throw new \BadMethodCallException('The menu manipulator must be set to get the breadcrumbs array');
+        }
+
+        $menu = $this->castMenu($menu);
+
+        return $this->menuManipulator->getBreadcrumbsArray($menu, $subItem);
+    }
+
+    /**
+     * Checks whether an item is current.
+     *
+     * @param ItemInterface $item
+     *
+     * @return boolean
+     */
+    public function isCurrent(ItemInterface $item)
+    {
+        if (null === $this->matcher) {
+            throw new \BadMethodCallException('The matcher must be set to check if item is current');
+        }
+
+        return $this->matcher->isCurrent($item);
```

Comment:
Ì would rather inject the Matcher directly in the Twig extension instead of adding this method here. It would save a method call each time